### PR TITLE
Tadams/7132/scaffold user service modules

### PR DESCRIPTION
**Pull request by StarChaser44**
Completes story https://app.shortcut.com/clarkcan/story/7132/scaffold-user-service-modules

### CODE REVIEW

File: test_environment/data.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+const { ObjectID, ObjectId } = require("mongodb");
+
+// mocks
+const SEED_OBJECTS = {
```
Comment (EVogel1999) [inline test_environment/data.js:4]: For this data.js we don't need any of the resource mocks.  Just an empty object

**Review by EVogel1999 — COMMENTED** [review]
One small change for the data.js file in the test_environment folder.  Other than that looks good!  Thank you!

**Review by pzalep1 — COMMENTED** [review]
After Emily's comment is addressed this looks good!

File: test_environment/functions.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import supertest from 'supertest';
```
Comment (JosueMolinaMorales) [inline test_environment/functions.js:1]: Is it possible to rename the file from functions.js to something else descriptive?

**Review by JosueMolinaMorales — COMMENTED** [review]
Just 1 comment. Looks good though

File: test_environment/functions.js (hunk 2 above)
Comment (StarChaser44) [reply test_environment/functions.js:1]: What do you have in mind?

Comment (JosueMolinaMorales) [reply test_environment/functions.js:1]: setUpTest.js or setUpSuperTest.js

**Review by EVogel1999 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**